src/FSharp.Core/local.fsi

```diff
@@ -6,22 +6,22 @@ open Microsoft.FSharp.Core
 
 [<AutoOpen>]
 module internal DetailedExceptions =
-    val inline invalidArgFmt: arg: string -> format: string -> paramArray: obj array -> _
```

Comment:
So will using underscore like here, break determinism together with graph based tc?

Reply:
Yes, because the signature will produce a name with a generated id (see screenshot).

Reply:
Right, so we should probably have at least a tracking issue to either: solve this by making this generation deterministic (regardless of compilation order) or produce some sort of warning from graph based tc when this construct is used?

Reply:
I think this construct is common-enough that we need to solve it anyway.